common/app/model/dotcomrendering/LinkedData.scala

```diff
@@ -117,6 +117,18 @@ object LinkedData {
     }
   }
 
+  def forInteractive(
+      article: InteractivePage,
+  ): List[LinkedData] = {
+    List(
+      WebPage(
+        `@id` = article.metadata.webUrl,
+        potentialAction =
+          PotentialAction(target = "android-app://com.guardian/" + article.metadata.webUrl.replace("://", "/")),
```

Comment:
Question (needs answered before merging): Do we want to direct users to the android app (when applicable) for interactives? Does the app support them or is there a risk that they just end up getting re-directed back into the web

Reply:
ping @MarSavar @JamieB-gu?

Reply:
If this will be something to discuss, maybe we should take it out this PR?

Reply:
@jamesgorrie Updated PR to remove potentialAction for the time being!

Reply:
> Does the app support them

Yes; the app should support all the same articles from CAPI that dotcom does.